CONTRIBUTING.md

```diff
@@ -0,0 +1,120 @@
+# Contributing to dGrants
+
+A big welcome and thank you for considering contributing to dGrants open source projects! It’s people like you that make it a reality for users in our community.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+## Quicklinks
+
+* [Code of Conduct](#code-of-conduct)
+* [Getting Started](#getting-started)
+    * [Discussions](#discussions)
+    * [Issues](#issues)
+    * [Pull Requests](#pull-requests)
+      * [Git Rebase Workflow](#git-rebase-workflow)
+* [Getting Help](#getting-help)
+
+## Code of Conduct
+
+Contributions to Gitcoin are governed by the [Contributor Covenant version 1.4](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html).
+All contributors and participants agree to abide by its terms. To report
+violations, shoot out an email to founders@gitcoin.co
+
+The Code of Conduct is designed and intended, above all else, to help establish
+a culture within the project that allows anyone and everyone who wants to
+contribute to feel safe doing so.
+
+Open, diverse, and inclusive communities live and die on the basis of trust.
+Contributors can disagree with one another so long as they are done in good
+faith and everyone is working towards a common goal.
+
+## Getting Started
+
+Contributions are made to this repo via Discussions, Issues and Pull Requests (PRs). A few general guidelines that cover the trio:
+
+- Search for existing Issues and PRs before creating your own.
+- We work hard to makes sure issues are handled in a timely manner but, depending on the impact, it could take a while to investigate the root cause. A friendly ping in the comment thread to the submitter or a contributor can help draw attention if your issue is blocking.
+
+### Discussions
+
+Discussion should be used to discuss potential changes before a PR is created,  or discussing the implmentation on a current feature. Once the discussion has ended with participation from the community and maintainers, if changes need to made to the codebase, follow it up by creating an issue.
```

Comment:
IIRC we agreed the flow would almost always be discussion > issue > PR, so just clarifying that here. Also fixing some typos

```suggestion
Discussion should be used to discuss potential changes before an issue or PR is created, or discussing the implementation on a current feature. Once the discussion has ended with participation from the community and maintainers, if changes need to made to the codebase, follow it up by creating an issue.
```